src/HistoryService.Mappers/Responses/Interfaces/IFindServiceResponseMapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.HistoryService.Models.Db;
+using LT.DigitalOffice.HistoryService.Models.Dto.Responses;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.HistoryService.Mappers.Responses.Interfaces
+{
+    [AutoInject]
+    public interface IFindServiceResponseMapper
+    {
+        public ServiceInfo Map(DbService dbService);
```

Comment:
remove public

Reply:
name of mapper doesn't match return type

Reply:
fixed